modules/spo-creating-profiles.adoc

```diff
@@ -20,7 +20,7 @@ endif::[]
 
 Use the `{kind}` object to create profiles.
 
-ifdef::seccomp[]
+ifdef::spo-seccomp[]
 
 `{kind}` objects can restrict syscalls within a container, limiting the access of your application.
```

Comment:
Same for this section, maybe the preview needs to be updated?

Reply:
@abrennan89 Oops, yes, absolutely good catch. Doing that now!

Reply:
This has been done!